safehttp/plugins/csp/csp_test.go

```diff
@@ -230,3 +233,119 @@ func TestNonceEmptyContext(t *testing.T) {
 		t.Errorf("Nonce(ctx) got nonce: %v want: %v", n, want)
 	}
 }
+
+func TestCommitNonce(t *testing.T) {
+	rec := safehttptest.NewResponseRecorder()
+	req := safehttptest.NewRequest(safehttp.MethodGet, "https://foo.com/pizza", nil)
+	req.SetContext(context.WithValue(req.Context(), ctxKey{}, "pizza"))
+
+	it := Interceptor{}
+	tr := safehttp.TemplateResponse{FuncMap: map[string]interface{}{}}
+	it.Commit(rec.ResponseWriter, req, tr, nil)
+
+	nonce, ok := tr.FuncMap["CSPNonce"]
+	if !ok {
+		t.Error(`tr.FuncMap["CSPNonce"]: got nil, want nonce`)
+	}
+
+	fn, ok := nonce.(func() string)
+	if !ok {
+		t.Errorf(`tr.FuncMap["CSPNonce"]: got %T, want "func() string"`, fn)
+	}
+	if want, got := "pizza", fn(); want != got {
```

Comment:
nit: Make the assignment inside the condition consistent with how you display it in tests, i.e. `got` before `want`.